test/e2e/cli_test.go

```diff
@@ -659,6 +677,46 @@ func (s *CLIWithServerSuite) TestWorkflowRetry() {
 		})
 }
 
+func (s *CLIWithServerSuite) TestRetryByLabelSelector() {
+	s.testNeedsOffloading()
+	var retryTime corev1.Time
+
+	s.Given().
```

Comment:
this feels a bit copy-and-paste - can you simplify it to only contain more basic assertions?

Reply:
I've completed a set of test all commands and all options with the basic assertion as you recommended. :)